advanced/index.php

```diff
@@ -57,16 +57,21 @@ function setHeader($type = "x") {
 } elseif (filter_var($serverName, FILTER_VALIDATE_IP) || in_array($serverName, $authorizedHosts)) {
     // Set Splash Page output
     $splashPage = "
-    <html>
-      <head>
-        $viewPort
-        <link rel='stylesheet' href='pihole/blockingpage.css' type='text/css'/>
-      </head>
-      <body id='splashpage'>
-        <img src='admin/img/logo.svg'/><br/>
-        Pi-<b>hole</b>: Your black hole for Internet advertisements<br/>
-        <a href='/admin'>Did you mean to go to the admin panel?</a>
-      </body>
+    <!doctype html>
+    <html lang='en'>
+        <head>
+            <meta charset='utf-8'>
+            $viewPort
+            <title>● $serverName</title>
+            <link rel='stylesheet' href='pihole/blockingpage.css'>
+            <link rel='shortcut icon' href='admin/img/favicons/favicon.ico' type='image/x-icon'>
+        </head>
+        <body id='splashpage'>
+            <img src='admin/img/logo.svg' alt='Pi-hole logo' width='256' height='377'>
```

Comment:
This reminds me, it seems to be broken when a not found page is shown since it needs to be `/admin...`.

Try for example, `http://IP/admin/foo`.

Let me know how to proceed @PromoFaux @DL6ER

Reply:
@PromoFaux @DL6ER for follow-up.

Reply:
To be honest, a lot of me wants the block page to die. It had it's purpose a few years ago, but with the ever increasing uptake of https, the page serves very little purpose (aside from for redirecting from `pi.hole` to `pi.hole/admin`

Reply:
You're not alone in that view.

Reply:
I wonder if we should just remove it for 5.1, at least for new installs. We could update the installer to not overwrite anything that is already in the `/var/www/html/pihole`, and keep it in the repo as a starting point for anyone that _really_ wants make use of a custom block page. /shrug

Reply:
BTW this must be #3493

Reply:
I don't use this feature myself, so if you guys think it's better to drop it, I trust you better. That being said, this PR should streamline some stuff. If we decide to drop it, I can rebase #3425 to not depend on this PR.

Reply:
I would be fine with dropping it completely but I think that's something we need to get community feedback from first. It will affect users and in some cases it is used.

Reply:
There will be users who still use it. But as Promo suggested removing it for new installs and leave old untouched. might be a compromise.

Reply:
I'm not sure how we will handle the blockpage in version 6. The page now is just a glorified 404 error page and really it's a hacky way of doing things. I guess we could have PH7/civetweb display a blockpage but the idea with version 6 is that we no longer use port 80. (Using port 443 is an entirely different discussion.) 

So there's a good chance it will go away for that reason, and it would be nice to not step on real 404 errors...

Reply:
In the meantime, I've tested these changes and the block page still works as it used to, so.. I will approve for now. 

We can continue the conversation when we're closer to discussion 6.0 in detail I guess!